orchestrator/src/main/java/com/yahoo/vespa/orchestrator/model/StorageNodeImpl.java

```diff
@@ -69,41 +62,7 @@ public void setNodeState(OrchestratorContext context, ClusterControllerNodeState
                 ", node index " + nodeIndex +
                 ", node state " + wantedNodeState);
 
-        ClusterControllerStateResponse response;
-        try {
-            response = client.setNodeState(context, nodeIndex, wantedNodeState);
-        } catch (IOException e) {
-            throw new HostStateChangeDeniedException(
-                    hostName(),
-                    HostedVespaPolicy.CLUSTER_CONTROLLER_AVAILABLE_CONSTRAINT,
-                    "Failed to communicate with cluster controllers " + clusterControllers + ": " + e,
-                    e);
-        } catch (WebApplicationException e) {
-            Response webResponse = e.getResponse();
-            // Response may contain detail message
-            ClusterControllerStateErrorResponse errorResponse = webResponse.readEntity(ClusterControllerStateErrorResponse.class);
-            String detail = errorResponse.message == null ? "" : ": " + errorResponse.message;
-            throw new HostStateChangeDeniedException(
-                    hostName(),
-                    HostedVespaPolicy.SET_NODE_STATE_CONSTRAINT,
-                    "Failure from cluster controllers " + clusterControllers + " when setting node " + nodeIndex +
-                            " in cluster " + clusterId + " to state " + wantedNodeState + detail,
-                    e);
-        } catch (UncheckedTimeoutException e) {
-            throw new HostStateChangeDeniedException(
-                    hostName(),
-                    HostedVespaPolicy.DEADLINE_CONSTRAINT,
-                    "Timeout while waiting for setNodeState(" + nodeIndex + ", " + wantedNodeState +
-                            ") against " + clusterControllers + ": " + e.getMessage(),
-                    e);
-        }
-
-        if ( ! response.wasModified) {
-            throw new HostStateChangeDeniedException(
-                    hostName(),
-                    HostedVespaPolicy.SET_NODE_STATE_CONSTRAINT,
-                    "Failed to set state to " + wantedNodeState + " in cluster controller: " + response.reason);
-        }
+        client.setNodeState(context, storageService.hostName(), nodeIndex, wantedNodeState);
```

Comment:
If a probe was denied, this method used to throw an exception which was important.

Reply:
Oh, looked to me like this was never a probe. Ok, fine.